mcs/class/System.Security/Makefile

```diff
@@ -8,7 +8,7 @@ MONO_SECURITY=Mono.Security
 endif
 
 LIBRARY = System.Security.dll
-LIB_REFS = secxml/System bare/System.Xml $(MONO_SECURITY)
+LIB_REFS = secxml/System bare/System.Xml System.Numerics $(MONO_SECURITY)
```

Comment:
You cannot do that, System.Numerics is built after System.Security. What code does require it?